.github/workflows/rust.yml

```diff
@@ -18,26 +18,16 @@ jobs:
       matrix:
         runs-on: [ubuntu-22.04, windows-2022, macos-13]
         toolchain:
-          - "1.48"
+          - "1.61"
           - stable
           - nightly
         versions:
           - ""
           - "-Zminimal-versions"
-        exclude:
-          # `windows-core >= 0.51` uses `edition = "2021"`, but users of the library can can use
-          # `windows-core == 0.50` to be rust 1.48 compatible.
-          - runs-on: windows-2022
-            toolchain: "1.48"
-            versions: ""
```

Comment:
we haven't kept up to date with `windows-core` releases. Upstream started programatically tracking msrv here:

https://github.com/microsoft/windows-rs/blob/50009500b686f542530515aedba8334dc5b617b0/.github/workflows/msrv.yml#L37-L40

```yaml
      - name: Rust version
        run: rustup update --no-self-update 1.74 && rustup default 1.74
      - name: Check windows-core
        run:  cargo check -p windows-core --all-features
```

But this isn't historical and I'm not sure what range we should be using here. I filed https://github.com/strawlab/iana-time-zone/issues/158 for tracking this.